packages/fhir-converter/src/templates/cda/Sections/History_of_Present_Illness.hbs

```diff
@@ -7,6 +7,10 @@
                 {{/with}}
 
                 {{!-- TODO need to add references, author, etc --}}
+                {{#if ../../msg.ClinicalDocument.componentOf.encompassingEncounter}} 
```

Comment:
nit: generally we have been trying to stay away from using ".." notation to access variables. It is a bit confusing and makes it hard to track variable access. In this case though probably fine but just fyi for the future

Reply:
Hmm. Sounds good, I'll keep that in mind